packages/unpublished-model-encapsulation-test-app/package.json

```diff
@@ -18,6 +18,7 @@
     "test": "ember test --test-port=0"
   },
   "dependencies": {
+    "@ember-data/private-build-infra": "3.17.0-alpha.1",
```

Comment:
Do you need to add `ember-data/canary-features` in devDependencies at least?

Reply:
we don't have to, as it is a transitive dep of all of these other things, but making it explicit can't hurt